Update MCP Swift SDK to work with Swift 6. Fix server not terminating on SIGTERM.

diff --git a/Package.resolved b/Package.resolved
index 3459c90..9059972 100644
--- a/Package.resolved
+++ b/Package.resolved
@@ -1,5 +1,5 @@
 {
-  "originHash" : "90fc2bf5f6bc4eccbfa35d5ca20b54fc098142566350380bf3fc6537dfe977da",
+  "originHash" : "b768a5b6603d8bcedd61a7736c4b3e4ab78a2a2c3e60559e704da915f6010ed5",
   "pins" : [
     {
       "identity" : "eventsource",
@@ -15,8 +15,17 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/apple/swift-async-algorithms.git",
       "state" : {
-        "revision" : "6c050d5ef8e1aa6342528460db614e9770d7f804",
-        "version" : "1.1.1"
+        "revision" : "9d349bcc328ac3c31ce40e746b5882742a0d1272",
+        "version" : "1.1.3"
+      }
+    },
+    {
+      "identity" : "swift-atomics",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/apple/swift-atomics.git",
+      "state" : {
+        "revision" : "b601256eab081c0f92f059e12818ac1d4f178ff7",
+        "version" : "1.3.0"
       }
     },
     {
@@ -37,13 +46,22 @@
         "version" : "1.9.1"
       }
     },
+    {
+      "identity" : "swift-nio",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/apple/swift-nio.git",
+      "state" : {
+        "revision" : "57c0a08a331aaea9f5d7a932ad94ef43be942a95",
+        "version" : "2.100.0"
+      }
+    },
     {
       "identity" : "swift-sdk",
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/modelcontextprotocol/swift-sdk.git",
       "state" : {
-        "revision" : "c0407a0b52677cb395d824cac2879b963075ba8c",
-        "version" : "0.10.2"
+        "revision" : "a0ae212ebf6eab5f754c3129608bc5557637e605",
+        "version" : "0.12.1"
       }
     },
     {
diff --git a/Sources/Apple-Docs-MCP/Apple_Docs_MCP.swift b/Sources/Apple-Docs-MCP/Apple_Docs_MCP.swift
index 18c8ad5..b5b1809 100644
--- a/Sources/Apple-Docs-MCP/Apple_Docs_MCP.swift
+++ b/Sources/Apple-Docs-MCP/Apple_Docs_MCP.swift
@@ -31,7 +31,6 @@ struct Apple_Docs_MCP {
 
         serviceGroup = ServiceGroup(
             services: [mcpService],
-            gracefulShutdownSignals: [.sigterm, .sigint],
             logger: logger
         )
 
